libs/platform/src/lib/components/dynamic-page/dynamic-page-content/dynamic-page-content.component.html

```diff
@@ -0,0 +1,8 @@
+<ng-container *ngIf="tabLabel">
+    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
+</ng-container>
+
+<ng-container *ngIf="!tabLabel">
+    <ng-container *ngTemplateOutlet="contentTemplate"></ng-container>
```

Comment:
we are doing same thing in if and else statement. Do we need *ngIf in this case.

Reply:
We need this because our `fdp-dynamic-page-content` can take normal content as well as tabbed content where each tab's content is its own `fdp-dynamic-page-content`. Internally, we map it to `fdp-dynamic-page-tabbed-content` to get the individual tab projections. Just doing `select="fdp-dynamic-page-tabbed-content"` won't work as it will put all tab contents in a single tab's content.

https://stackoverflow.com/questions/44699469/how-to-use-multiple-ng-content-in-the-same-component-in-angular-2 explains more about this usage.

Reply:
👍